regulations/static/regulations/css/scss/partials/_sidebar_right.scss

```diff
@@ -166,11 +162,21 @@ aside.right-sidebar {
 }
 
 .related-rule-title {
-    color: $primary_link_color;
     text-decoration: none;
     font-size: $font-size-sm;
-}
 
+    &:link .link-heading::after {
```

Comment:
link and hover styles for the new home of the external link icon

https://css-tricks.com/snippets/css/link-pseudo-classes-in-order/